packages/cozy-harvest-lib/src/components/VaultCiphersList/index.jsx

```diff
@@ -30,9 +30,14 @@ class VaultCiphersList extends React.Component {
         type: CipherType.Login,
         uri: konnectorURI
       })
-      this.setState({
-        ciphers
-      })
+
+      if (ciphers.length > 0) {
+        this.setState({
+          ciphers
+        })
+      } else {
+        this.props.onNoCiphers()
```

Comment:
Second point : La fonction onNoCiphers a l'air optionnelle (au pire ça affiche le bouton "connecter un autre compte") et ça me parait très bien comme ça. Ca mériterait un code un peu défensif : 

```suggestion
        if (this.props.onNoCiphers) { this.props.onNoCiphers() }
```